spec/models/degree/what_degree_class_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Degree::WhatDegreeClass do
+  let(:what_degree_class) { build(:degree_what_degree_class) }
+  let(:wrong_answer) { build(:degree_what_degree_class, degree_class: "dont know") }
+
+  describe "validation" do
+    context "with invalid subject options" do
+      ['skiing', 'fishing', 'golfing', 'surfing' ].each do |invalid_subject|
```

Comment:
With these I don't think we need to test so many variants -- there's no benefit in testing multiple here

Reply:
Actually I was thinking as the all the  degree and equivalent specs are covered by their parent class/factory (e.g) we could get rid of all the "validation" specs ?

Reply:
Looking at this again, I pulled this test structure from one of the other DFE apps, it gives a false positive!